### PR TITLE
[UI] Use new core UI components

### LINKED ISSUE #3060: Replace some UI components with the existing ones in core-ui
**Component**: ui

**Why this is needed**:
We extracted/improved some common components in core-ui. 
Thanks to #3046 bump the core-ui version to 0.7.0, we should be able to benefit all the new core-ui features/components in MetalK8s UI.

**What should be done**:
- The health severity selector
image

- The number input component
image

- Enable the hoverable sidebar

**Implementation proposal** (strongly recommended):

**Test plan**:

### PR DESCRIPTION
**Pull request by alexis-ld**
**Component**:
ui, sidebar, inputs, alerts

**Context**: 

We want to use the new Core-UI components. #3060 

**Summary**:
* Replaced ActiveAlertsFilters by core-ui's HealthSelector.
* Made Sidebar hoverable.
* #3060 also mentions number inputs but those work without any adaptations

**Important** : For this to work we need this fix (https://github.com/scality/core-ui/pull/234) to be included in the core-ui version used by metalk8s.

**Acceptance criteria**: 
* Sidebar is now hoverable
* Number inputs are working
* Alerts are filtered through the Healthselector component

---

Closes: #3060

### CODE REVIEW

File: ui/src/components/ActiveAlertsFilters.js [hunk 1]
```diff
@@ -41,65 +17,42 @@ const ActiveAlertsFilter = (props) => {
...
-      <Dropdown
-        items={items}
-        text={dropDownLabel}
-        size="small"
-        data-cy="alert_filter"
-      />
-    </ActiveAlertsFilterWrapper>
+    <div data-cy="alert_filter">
+      <Healthselector items={items} />
+    </div>
```
Comment (ChengYanJin) [inline ui/src/components/ActiveAlertsFilters.js]: In my opinion, we should add the `data-cy` directly to `<Healthselector/>` so we don't need to adjust the code in the test.

**Review by ChengYanJin — CHANGES_REQUESTED** [review]
Was mainly focus on checking the test before. 
Code review, small change request.

File: ui/src/components/ActiveAlertsFilters.js (hunk 1 above)
Comment (alexis-ld) [reply ui/src/components/ActiveAlertsFilters.js]: @ChengYanJin That's what I did initially but seems `<Healthselector/>` is directly exported from core-ui the custom HTML attributes such as `data-cy` are not forwarded, that's why the tests were failing. Do you have an other way around that ? :)

Comment (ChengYanJin) [reply ui/src/components/ActiveAlertsFilters.js]: @alexis-ld  one thing we can do is to add the spread syntax to expand the propery of the component in core-ui.
```js
function Healthselector({ size, items, ...rest }: Props) {
  return (
    <HealthselectorContainer className="sc-healthselector" {...rest}>
}
```

**Review by ChengYanJin — APPROVED** [review]

**Pull request merged**